### ISSUE TITLE
Change is hard, automating web accessibility isn't.

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#606]
## Learning how to integrate accessibility testing into your existing processes is easier than ever with new features from WorldSpace Attest.

[Join us on July 12th, from 1:30-2:00PM EST](https://accessibility.deque.com/change-is-hard-automating-web-accessibility-isnt), as Noah Mashni, Solutions Architect at Deque, walks you through an introduction to Attest, Deque’s enterprise, automated accessibility testing tool.

WorldSpace Attest is a toolkit for running accessibility tests while you code. It equips your development team to run accessibility tests while they create new code and components. Accessibility issues will simply become part of functional issue reporting.

In this webinar, you will learn about:
- Attest Browser Extension - Quickly run in-browser tests that output detailed information on accessibility violations and how to fix them. Unlike the free extension axe, Attest allows you to export issue reports to your team.
- *new* Aget (Attest Get-er) - perfect for teams that prefer a lightweight, data-driven user flow that allows them to integrate accessibility testing into their build without pre-existing automated tests. The Aget Codeless Testing feature allows for quick testing via a config file of URLS and natural language command line.
- End-to-end integration testing demonstration of Attest
- Continuous Integration with Jenkins
- *new*Custom Dashboards - track and export accessibility project progress for leadership teams with these new Reports and Dashboards.

**Comment by ericwbailey** [comment]
All set, thank  you!
